LPS-72931 Build files

csierra opened the pull request:
@topolik has also reviewed and squashed his commits
/cc @marianoalvarosaiz @JavierMoral

JavierMoral committed:
LPS-72931 Added configuration to store LDAP server priority

Commit:
LPS-72931 Assign the order to each LDAPServerConfiguration

when they are saved in the form

Commit:
LPS-72931 Simplify

Commit:
LPS-72931 Return LDAPServerConfigurations in order

Commit:
LPS-72931 Build files

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/calendar/.gitrepo'.

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/952/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/952/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Some tests FAILED.</h1><p>Build Time: 1 hour 36 minutes 2 seconds 888 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f3ef22495f809d94954a1ad9c01468c885b48e60">f3ef22495f809d94954a1ad9c01468c885b48e60</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/952/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/133136/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131878/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132789/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115853/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123743/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tcserver32-mysql56-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131892/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tcserver32-mysql56-jdk8/1</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124203/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-db2105-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132438/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-db2105-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132439/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75395/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123938/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129129/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131885/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/82599/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107879/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132441/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75399/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-sybase160-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123939/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-sybase160-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75400/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107880/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123749/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-websphere85-mysql56-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131886/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-websphere85-mysql56-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132442/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107887/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/1</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123943/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118488/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115859/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111271/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/10</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129258/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/11</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118489/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/12</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124209/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/13</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111273/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/14</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115862/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/15</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124208/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/16</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118491/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/17</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129257/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/18</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107883/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/19</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129255/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/110748/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/20</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115864/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/21</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111274/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/22</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115863/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/23</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124210/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/24</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118492/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/25</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129259/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/26</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/110747/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/27</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107884/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/28</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/112799/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/29</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124206/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118503/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/30</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/112800/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/31</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124211/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/32</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115865/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/33</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/65015/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/34</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123944/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/35</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123753/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/36</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107885/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/37</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111275/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/38</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/122971/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/39</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111272/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/4</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118494/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/40</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/82604/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/41</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/82603/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/42</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115860/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/5</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124207/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/129256/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118490/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/115861/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/9</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118498/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131888/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/133146/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118499/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75403/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/65016/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118500/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/111279/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75404/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132796/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72308/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72307/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75407/">test-portal-acceptance-pullrequest-batch(master)/integration-db2105-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131891/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132798/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/133147/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle121-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/130526/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72306/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase160-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75406/">test-portal-acceptance-pullrequest-batch(master)/javadoc-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131890/">test-portal-acceptance-pullrequest-batch(master)/lpkg-override-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132797/">test-portal-acceptance-pullrequest-batch(master)/lpkg-persistence-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/133148/">test-portal-acceptance-pullrequest-batch(master)/lpkg-release-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132446/">test-portal-acceptance-pullrequest-batch(master)/lpkg-startup-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/130525/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75405/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131889/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/72305/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/133149/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle121-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/130527/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132445/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/65017/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132799/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118502/">test-portal-acceptance-pullrequest-batch(master)/portal-frontend-js-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/122976/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/107886/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/82607/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/112801/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/124212/">test-portal-acceptance-pullrequest-batch(master)/source-format-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/123945/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/122975/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/118501/">test-portal-acceptance-pullrequest-batch(master)/wsdd-builder-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/564/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li></ul><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/952/jenkins-report.html">here</a>.</h5><hr/><h4>Failed Jobs:</h4><ol><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/952/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>108 Jobs Passed.<br/>3 Jobs Failed.</p></div><pre><code>Downstream jobs FAILED.</code></pre></div></li><li><div><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/131889/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</a></h5><div><h6>Job Results:</h6><p>6526 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/131889/">AXIS_VARIABLE=3,label_exp=!master #131889</a><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/131889//testReport/com.liferay.asset.publisher.upgrade.v1_0_0.test/UpgradePortletPreferencesTest/testUpgradeDLDateFieldsValues">UpgradePortletPreferencesTest.testUpgradeDLDateFieldsValues</a><pre><code>java.lang.AssertionError: Thread Thread[liferay/scheduled_user_ldap_import-1,5,main] caught concurrent failure: java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.kernel.messaging.SerialDestination, message=Unable to process message {destinationName=liferay/scheduled_user_ldap_import, response=null, responseDestinationName=null, responseId=null, payload=null, values={GROUP_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener, companyId=0, groupId=0, DESTINATION_NAME=liferay/scheduled_user_ldap_import, EXCEPTIONS_MAX_SIZE=0, JOB_STATE=com.liferay.portal.kernel.scheduler.JobState@117ea4e, STORAGE_TYPE=MEMORY_CLUSTERED, JOB_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener}}
java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.kernel.messaging.SerialDestination, message=Unable to process message {destinationName=liferay/scheduled_user_ldap_import, response=null, responseDestinationName=null, responseId=null, payload=null, values={GROUP_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener, companyId=0, groupId=0, DESTINATION_NAME=liferay/scheduled_user_ldap_import, EXCEPTIONS_MAX_SIZE=0, JOB_STATE=com.liferay.portal.kernel.scheduler.JobState@117ea4e, STORAGE_TYPE=MEMORY_CLUSTERED, JOB_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener}}
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.j...</code></pre></div></li><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/131889//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/132445/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</a></h5><div><h6>Job Results:</h6><p>6526 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/132445/">AXIS_VARIABLE=3,label_exp=!master #132445</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/132445//testReport/com.liferay.asset.publisher.upgrade.v1_0_0.test/UpgradePortletPreferencesTest/testUpgradeDLDateFieldsValues">UpgradePortletPreferencesTest.testUpgradeDLDateFieldsValues</a><pre><code>java.lang.AssertionError: Thread Thread[liferay/scheduled_user_ldap_import-1,5,main] caught concurrent failure: java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.kernel.messaging.SerialDestination, message=Unable to process message {destinationName=liferay/scheduled_user_ldap_import, response=null, responseDestinationName=null, responseId=null, payload=null, values={GROUP_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener, companyId=0, groupId=0, DESTINATION_NAME=liferay/scheduled_user_ldap_import, EXCEPTIONS_MAX_SIZE=0, JOB_STATE=com.liferay.portal.kernel.scheduler.JobState@1285dce, STORAGE_TYPE=MEMORY_CLUSTERED, JOB_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener}}
java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.kernel.messaging.SerialDestination, message=Unable to process message {destinationName=liferay/scheduled_user_ldap_import, response=null, responseDestinationName=null, responseId=null, payload=null, values={GROUP_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener, companyId=0, groupId=0, DESTINATION_NAME=liferay/scheduled_user_ldap_import, EXCEPTIONS_MAX_SIZE=0, JOB_STATE=com.liferay.portal.kernel.scheduler.JobState@1285dce, STORAGE_TYPE=MEMORY_CLUSTERED, JOB_NAME=com.liferay.portal.security.ldap.internal.messaging.UserImportMessageListener}}
	at com.liferay.portal.test.rule.LogAssertionAppender.append(LogAssertionAppender.j...</code></pre></div></li><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/132445//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li></ol><h5>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h5></html>

Commit:
LPS-72931 Extra check for deletion

Configurations might be asynchronously deleted. Alas there seems to be
no way of knowing this in advance.

liferay-continuous-integration commented:
Closing and locking pull request because pull requests sent to this user may not be updated. Please resend this pull request.
